src/h5web/vis-packs/core/shared/VisCanvas.module.css

```diff
@@ -13,7 +13,6 @@
 
 /* R3F's root element */
 .canvasWrapper {
-  composes: no-invert-canvas from global;
```

Comment:
This was actually inverting back the line vis as well, which I don't think we want. Now the `data-keep-canvas-colors` is set on the `HeatmapVis`' wrapping `figure` element.